src/Pages/sunkyungnoh/Main/components/Comments.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import Comment from './comment';
+import './Comments.scss';
+
+class Comments extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      commentArr: [],
+      newComment: '',
+      value: '',
+    };
+  }
+
+  componentDidMount() {
+    this.setState({ commentArr: this.props.prevComment });
+  }
+
+  addComment = e => {
+    e.preventDefault();
+    const { commentArr, newComment } = this.state;
+    this.setState({
+      commentArr: [
+        ...commentArr,
+        {
+          id: commentArr.length + 1,
+          userName: '펭수',
+          content: newComment,
+          isLiked: false,
+        },
+      ],
+      value: '',
+    });
+  };
+
+  render() {
+    const { commentArr, newComment, value } = this.state;
+    return (
+      <div className="comments">
+        <ul className="commentList">
+          {commentArr.map(comment => (
+            <Comment
+              key={comment.id}
+              userName={comment.userName}
+              content={comment.content}
+              isLiked={comment.isLiked}
+            />
+          ))}
+        </ul>
+        <form onSubmit={this.addComment}>
+          <input
+            onChange={e => {
+              this.setState({ newComment: e.target.value });
+              this.setState({ value: e.target.value });
+            }}
```

Comment:
```suggestion
            onChange={e => {
              this.setState({
                newComment: e.target.value,
                value: e.target.value
            }}
```
- setState 한번으로 업데이트 가능합니다
- state 키값은 다른데 같은 데이터를 넣어주고 있습니다. 용도가 궁금하네요! 🤔

Reply:
newComment가 새로운 댓글로 올라가고, input에 입력값은 초기화 시키고 싶어서 따로 주었었는데,,! 수정했습니다!